scripts/deploy-ces-goerli.sh

```diff
@@ -105,12 +99,6 @@ seth send "$RWA_JOIN" 'deny(address)' "$ETH_FROM"
 [[ -z "$MIP21_LIQUIDATION_ORACLE" ]] && {
     MIP21_LIQUIDATION_ORACLE=$(dapp create RwaLiquidationOracle "$MCD_VAT" "$MCD_VOW")
 
-    seth send "$MIP21_LIQUIDATION_ORACLE" 'init(bytes32,uint256,string,uint48)' \
-        "$ILK_ENCODED" \
-        "$MIP21_LIQUIDATION_ORACLE_INIT_VAL" \
-        "$MIP21_LIQUIDATION_ORACLE_INIT_DOC" \
-        "$MIP21_LIQUIDATION_ORACLE_INIT_TAU"
-
```

Comment:
We probably don't want to remove this in the deploy script for our own fork.

Reply:
Spell contains this initialization of LiquidationOracle